lua/wikis/counterstrike/Infobox/Person/Player/Custom.lua

```diff
@@ -21,35 +21,11 @@ local Widgets = require('Module:Widget/All')
 local Cell = Widgets.Cell
 
 local BANNED = mw.loadData('Module:Banned')
-local ROLES = {
-	-- Players
-	['awper'] = {category = 'AWPers', display = 'AWPer', store = 'awp'},
-	['igl'] = {category = 'In-game leaders', display = 'In-game leader', store = 'igl'},
-	['lurker'] = {category = 'Riflers', display = 'Rifler', category2 = 'Lurkers', display2 = 'lurker'},
-	['support'] = {category = 'Riflers', display = 'Rifler', category2 = 'Support players', display2 = 'support'},
-	['entry'] = {category = 'Riflers', display = 'Rifler', category2 = 'Entry fraggers', display2 = 'entry fragger'},
-	['rifler'] = {category = 'Riflers', display = 'Rifler'},
-
-	-- Staff and Talents
-	['analyst'] = {category = 'Analysts', display = 'Analyst', coach = true},
-	['broadcast analyst'] = {category = 'Broadcast Analysts', display = 'Broadcast Analyst', talent = true},
-	['observer'] = {category = 'Observers', display = 'Observer', talent = true},
-	['host'] = {category = 'Hosts', display = 'Host', talent = true},
-	['journalist'] = {category = 'Journalists', display = 'Journalist', talent = true},
-	['expert'] = {category = 'Experts', display = 'Expert', talent = true},
-	['producer'] = {category = 'Production Staff', display = 'Producer', talent = true},
-	['director'] = {category = 'Production Staff', display = 'Director', talent = true},
-	['executive'] = {category = 'Organizational Staff', display = 'Executive', management = true},
-	['coach'] = {category = 'Coaches', display = 'Coach', coach = true},
-	['assistant coach'] = {category = 'Coaches', display = 'Assistant Coach', coach = true},
-	['manager'] = {category = 'Managers', display = 'Manager', management = true},
-	['director of esport'] = {category = 'Organizational Staff', display = 'Director of Esport', management = true},
-	['caster'] = {category = 'Casters', display = 'Caster', talent = true},
-}
-ROLES.awp = ROLES.awper
-ROLES.lurk = ROLES.lurker
-ROLES.entryfragger = ROLES.entry
-ROLES.rifle = ROLES.rifler
+
+local ContractRoles = Lua.import('Module:ContractRoles')
+local StaffRoles = Lua.import('Module:StaffRoles')
+local InGameRoles = Lua.import('Module:InGameRoles')
+local ROLES = Table.merge(ContractRoles, StaffRoles, InGameRoles)
```

Comment:
could likely move that to commons and just import 1 module here
(using `Lua.requireIfExists('Module:InGameRoles', {loadData = true})` for the ingame roles)